root/programs/C#/Samples/WebApp_sample/MVC_Sample/MVC_Sample/App_Start/BundleConfig.cs

```diff
@@ -31,37 +36,75 @@ public class BundleConfig
         /// </summary>
         public static void RegisterBundles(BundleCollection bundles)
         {
-            bundles.Add(new ScriptBundle("~/bundles/jquery").Include(
-                        "~/Scripts/jquery-{version}.js"));
-
-            bundles.Add(new ScriptBundle("~/bundles/jqueryui").Include(
-                        "~/Scripts/jquery-ui-{version}.js"));
-
-            bundles.Add(new ScriptBundle("~/bundles/jqueryval").Include(
-                        "~/Scripts/jquery.unobtrusive*",
-                        "~/Scripts/jquery.validate*"));
-
-            // 開発と学習には、Modernizr の開発バージョンを使用します。次に、実稼働の準備が
-            // できたら、http://modernizr.com にあるビルド ツールを使用して、必要なテストのみを選択します。
-            bundles.Add(new ScriptBundle("~/bundles/modernizr").Include(
-                        "~/Scripts/modernizr-*"));
-
-            bundles.Add(new StyleBundle("~/Content/css").Include("~/Content/site.css"));
-
-            bundles.Add(new StyleBundle("~/Content/themes/base/css").Include(
-                        "~/Content/themes/base/jquery.ui.core.css",
-                        "~/Content/themes/base/jquery.ui.resizable.css",
-                        "~/Content/themes/base/jquery.ui.selectable.css",
-                        "~/Content/themes/base/jquery.ui.accordion.css",
-                        "~/Content/themes/base/jquery.ui.autocomplete.css",
-                        "~/Content/themes/base/jquery.ui.button.css",
-                        "~/Content/themes/base/jquery.ui.dialog.css",
-                        "~/Content/themes/base/jquery.ui.slider.css",
-                        "~/Content/themes/base/jquery.ui.tabs.css",
-                        "~/Content/themes/base/jquery.ui.datepicker.css",
-                        "~/Content/themes/base/jquery.ui.progressbar.css",
-                        "~/Content/themes/base/jquery.ui.theme.css",
-                        "~/Content/themes/base/jquery-ui.css"));
+            // see : https://www.asp.net/ajax/cdn
+
+            string jqueryVersion = "3.1.1";
+
+            BundleTable.EnableOptimizations = true;
+            BundleTable.Bundles.UseCdn = true; // same as: bundles.UseCdn = true;
+
+            // ( new ScriptBundle("~/XXXX") のパスは実在するpathと被るとRender時にバグる。
+            // なので、bundlesと実在しないpathを指定している（CSSも同じbundlesを使用する）。
+
+            bundles.Add(new ScriptBundle("~/bundles/app").Include(
+                        "~/Scripts/app/Site.js"));
+
+            bundles.Add(new ScriptBundle("~/bundles/otr").Include(
```

Comment:
@daisukenishino 

I think that there are few people who can understand that *otr* is an abbreviation for *Open Touryo*.
So, it might be a good idea to rename this path to *opentouryo*.